src/System.Linq.Expressions/src/System/Runtime/CompilerServices/DebugInfoGenerator.cs

```diff
@@ -20,7 +20,7 @@ public abstract class DebugInfoGenerator
         /// <returns>PDB symbol generator.</returns>
         public static DebugInfoGenerator CreatePdbGenerator()
         {
-            // Creating PDBs is not suported in .NET Core
```

Comment:
Hmm. There are quite a few things "suported" in the code base as a whole, and even more "unsuported". Submitting a PR for those outside of Expressions.